test/docker-compose.yml

```diff
@@ -101,15 +101,47 @@ services:
         condition: service_healthy
     restart: on-failure
 
-  nats:
+  nats-1:
     image: nats:2.10.10
-    container_name: 'nats-server'
-    restart: on-failure
-    expose:
-      - "4222"
+    container_name: nats-server-1
+    restart: no
```

Comment:
Why did you change `restart` option?

Reply:
@arkadiuszos4chain My idea was that the whole e2e test should fail if for any reason of misconfiguration of NATS stops and NATS shouldn't restart in that case. But there is not a strong reason to change it. Could also revert the change.

Reply:
I was just curious. Thanks for the clarification